src/utils/fishData.js

```diff
@@ -760,12 +728,6 @@ const fishComponents = {
       knnData: [],
       type: FishBodyPart.TAIL
     },
-    spikey2: {
-      index: 13,
-      src: 'images/fish/tailFin/Tail_Fin_Spikey2.png',
-      knnData: [],
-      type: FishBodyPart.TAIL
-    },
     square1: {
       index: 14,
```

Comment:
Do we need to clean up `index`es at some point?

Reply:
Yes. I want to wait on that for right now because I also want to rearrange some of these...

Reply:
Actually I have a change to remove all the indices so I'm going to merge this PR and send it out.